app/dash_app.py

```diff
@@ -1,383 +1,397 @@
-                x=dfCohFull['Reference Date'].unique(),
-                y=dfCohFull['Pair Date'].unique(),
-                color_continuous_scale='RdBu_r'
-                )
-fig.update_yaxes(autorange=True)
-
-app.layout = html.Div(id='parent', children=[
-    html.Div(style={'width': '5%', 'display': 'inline-block'}),
-    html.Img(src=app.get_asset_url(
-                'Seal_of_the_Geological_Survey_of_Canada.png'),
-             style={'width': '10%',
-                    'display': 'inline-block'}),
-    html.Div(style={'width': '5%',
-                    'display': 'inline-block'}),
-    html.H1(id='H1',
-            children='Volcano InSAR Interpretation Workbench',
-            style={'textAlign': 'center',
-                   'marginTop': 40,
-                   'marginBottom': 40,
-                   'display': 'inline-block'}),
-    html.Div(style={'height': '10px'}),
-    html.Div(style={'width': '5%',
-                    'display': 'inline-block'}),
-
-    html.Div(
-        dcc.Dropdown(stackList, stackList[1], id='site-dropdown'),
-        style={'width': '35%',
-               'display': 'inline-block',
-               'color': 'black'}),
-
-    html.Div(style={'width': '5%',
-                    'display': 'inline-block'}),
-
-    html.Div([
-        dcc.Graph(id='graph', figure=fig,
-                  style={'position': 'absolute',
-                         'width': '25%',
-                         'height': '45%',
-                         'margin-left': '69.5%',
-                         'margin-top': '0.5%',
-                         'zIndex': 2}
-                         ),
-        dl.Map([dl.TileLayer(),
-                dl.LayersControl(
-                    dl.BaseLayer(
-                            dl.TileLayer(
-                                # Basemaps
-                                url='https://basemap.nationalmap.gov/arcgis/rest/services/USGSTopo/MapServer/tile/{z}/{y}/{x}',
-                                attribution='Tiles courtesy of the <a href="https://usgs.gov/">U.S. Geological Survey</a>'
+    """Parse configuration from supplied file."""
+    config_obj = configparser.ConfigParser()
+    config_obj.read(args)
+    return config_obj
+
+
+config = get_config_params('config.ini')
+GEOSERVER_ENDPOINT = config.get('geoserver', 'geoserverEndpoint')
+
+# TODO add support for some or all of the following parameters to config
+
+# dashboard configuration
+TEMPLATE = 'darkly'
+TITLE = 'Volcano InSAR Interpretation Workbench'
+INITIAL_TARGET = 'Meager_5M3'
+
+# basemap configuration
+BASEMAP_URL = (
+    'https://basemap.nationalmap.gov/arcgis/rest/services/USGSTopo/MapServer'
+    '/tile/{z}/{y}/{x}')
+BASEMAP_ATTRIBUTION = (
+    'Tiles courtesy of the '
+    '<a href="https://usgs.gov/">U.S. Geological Survey</a>')
+BASEMAP_NAME = 'USGS Topo'
+
+# coherence plotting configuration
+GROUP_GAP_DAYS = 60
+CMAP_NAME = 'RdYlBu_r'
+COH_LIMS = (0.25, 0.65)
+
+# TODO read target configuration from database
```

Comment:
remove todo and create repo issue

Reply:
Issue #11 exists, and TODO can be removed when it is resolved.